src/server/app/util/schedulers/inactivity-notification.server.service.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+let path = require('path'),
+	q = require('q'),
+	deps = require(path.resolve('./src/server/dependencies.js')),
+	logger = deps.logger;
+
+let services = [
+	{
+		path: require(path.resolve('./src/server/app/util/services/inactivity-email.server.service.js')),
+		name: 'inactivity-notification'
```

Comment:
If this is meant only to handle inactive users, let's rename things to be more specific so that in the future, if we want to have notifications for inactive resources, we can add that more easily.